docs/src/guides/create-a-renderer/part-1-create-a-renderer.md

```diff
@@ -0,0 +1,149 @@
+## Part 1: Deploying a Renderer on Arweave
+
+In this tutorial, you'll learn how to deploy a renderer on the Arweave network that displays text content fetched from the permaweb. Additionally, you will deploy a "Hello, World!" text file using Bundlr and render it using the deployed renderer.
+
+### Prerequisites
+
+Before we begin, make sure you have the following:
+
+-   Basic knowledge of JavaScript, HTML, and CSS
+-   An Arweave wallet with a small amount of AR to deploy the renderer and the text file
+-   [Arweave Deploy CLI](https://github.com/ArweaveTeam/arweave-deploy) installed and configured
```

Comment:
Actually, I'm not sure why this is a prerequisite? It doesn't seem like it's used anywhere in the guide?

Reply:
Also you should not need AR to deploy a renderer if using bundlr-network

Reply:
I would recommend bundlr-network:

1 - its free for anything under 100k
2 - it is instant in terms of cache and comes with retry capability if it has trouble writing to the network
3 - it allows the network to scale 50k tps.

Reply:
Brilliant. I thought as much, but wasn't sure if I was perhaps missing something RE reccomending the most basic atomic method for the docs. Updating the PR now